### ISSUE TITLE
Renumbering reference sequence

### ISSUE THREAD

**Issue by aamcbride** [GMOD/jbrowse#639]
We need to renumber our reference sequence of an integrated viral genome from HPV18:1..7857 to HPV18:4001..7857/1..4000. 
See http://pave.niaid.nih.gov/#explore/epigenetics/chip_seq_on_integrated_hpv18_in_hela_cells
Can someone help advise us?
Thank you!

**Comment by cmdcolin** [comment]
Can you describe this a little more?

**Comment by aamcbride** [comment]
Yes. We have a circular HPV viral genome with very defined numbering 1-7857, which is integrated in the human genome at position 4000 in the virus. We'd like to be able to maintain the reference numbers of the original virus sequence in the J Browse display. So it should really be 4001-7857/1-4000. Is there a way to change the numbering display? The actual nucleotide sequence is correct. It's just that the numbers don't match the original virus reference numbers. Elements in the virus are named according to these numbers. For example, P105 is the early promoter at nucleotide 105. Thanks for your help.

**Comment by cmdcolin** [comment]
Would all the data from the bigwig tracks and everything need to be moved around? Just asking because http://github.com/GMOD/Apollo (which I work on) is testing out the ability to "project" features and sequences into different places.

**Comment by aamcbride** [comment]
I don't think that the tracks would have to be altered, they are displaying the correct information with respect to the sequence. I think a workaround would be to display the viral genes, as this would at least orient the person viewing the data, even if the actual numbering of the sequence didn't match what they are familiar with.




On Oct 14, 2015, at 5:08 PM, Colin Diesh <notifications@github.com<mailto:notifications@github.com>> wrote:


Would all the data from the bigwig tracks and everything need to be moved around? Just asking because http://github.com/GMOD/Apollo<https://github.com/GMOD/Apollo> (which I work on) is testing out the ability to "project" features and sequences into different places.

**Comment by rbuels** [comment]
How about a plugin that makes the display show *two* sets of coordinates, the coordinates on the regular reference, and then one with the coordinates of the virus sequence?
